bcdaworker/log/log.go

```diff
@@ -0,0 +1,24 @@
+package workerlog
```

Comment:
this doesn't need to be in its own package; I created these types so that they could be added to the context within bcdaworker. We could probably add this to one of the other log packages.